addons/account/account_view.xml

```diff
@@ -1701,10 +1701,22 @@
                     <group>
                         <group string="Amount Computation">
                             <field name="value" widget="radio" nolabel="1" colspan="4"/>
-                            <label for="value_amount" string="Amount To Pay"  attrs="{'invisible':[('value','=','balance')]}"/>
-                            <div attrs="{'invisible':[('value','=','balance')]}">
-                                <field name="value_amount" class="oe_inline"/>
-                            </div>
+                            <group>
+                                <div attrs="{'invisible':[('value','=','balance')]}">
+                                    <label for="value_amount" string="Enter a value"/>
+                                    <field name="value_amount" class="oe_inline"/>
+                                </div>
+                            </group>
+                            <group>
+                                <div attrs="{'invisible':[('value','=','balance')]}" style="margin-top: -18px">
```

Comment:
The only way I found to have everything well aligned like this : 
https://docs.google.com/file/d/0Bz6OpV442af-Z1hpbmkwM0psdTA/edit?usp=drivesdk
instead of this : 
https://docs.google.com/file/d/0Bz6OpV442af-QllOMnBsNU1aWnc/edit?usp=drivesdk
or other alignment with a bad render.

What should I do ? Remove the oe_inline ?